src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsFormForWizard.tsx

```diff
@@ -141,180 +140,178 @@ class RuleConditionsFormForWizard extends React.PureComponent<Props, State> {
-                  );
-                  return (
-                    <SelectControl
-                      value={mappedValue}
-                      inFieldLabel={t('Data Source: ')}
-                      onChange={optionObj => {
-                        const optionValue = optionObj.value;
-                        onChange(optionValue, {});
-                        onBlur(optionValue, {});
-                        // Reset the aggregate to the default (which works across
-                        // datatypes), otherwise we may send snuba an invalid query
-                        // (transaction aggregate on events datasource = bad).
-                        optionValue === 'transaction'
-                          ? model.setValue('aggregate', DEFAULT_TRANSACTION_AGGREGATE)
-                          : model.setValue('aggregate', DEFAULT_AGGREGATE);
-
-                        // set the value of the dataset and event type from data source
-                        const {dataset, eventTypes} =
-                          DATA_SOURCE_TO_SET_AND_EVENT_TYPES[optionValue] ?? {};
-                        model.setValue('dataset', dataset);
-                        model.setValue('eventTypes', eventTypes);
-                      }}
-                      options={dataSourceOptions}
-                      isDisabled={disabled}
-                      required
-                    />
-                  );
-                }}
-              </FormField>
-              <FormField
-                name="query"
-                inline={false}
-                style={{
-                  ...formElemBaseStyle,
-                  flex: '6 0 700px',
-                }}
-                flexibleControlStateSize
-              >
-                {({onChange, onBlur, onKeyDown, initialData, model}) => (
-                  <SearchContainer>
-                    <StyledSearchBar
-                      defaultQuery={initialData?.query ?? ''}
-                      omitTags={['event.type']}
-                      disabled={disabled}
-                      useFormWrapper={false}
-                      organization={organization}
-                      placeholder={
-                        model.getValue('dataset') === 'events'
-                          ? t('Filter events by level, message, or other properties...')
-                          : t('Filter transactions by URL, tags, and other properties...')
-                      }
-                      onChange={onChange}
-                      onKeyDown={e => {
-                        /**
-                         * Do not allow enter key to submit the alerts form since it is unlikely
-                         * users will be ready to create the rule as this sits above required fields.
-                         */
-                        if (e.key === 'Enter') {
-                          e.preventDefault();
-                          e.stopPropagation();
-                        }
+      <React.Fragment>
```

Comment:
Big diff here only caused by moving the form fields out of the `<Panel>` and changing the labels and min-width of a couple form fields

Reply:
You can use the ?w=1 (whitespace) option on github to hide whitespace diff
![image](https://user-images.githubusercontent.com/1421724/112882319-a8aca600-9081-11eb-9f6d-c10b626b19d0.png)

Reply:
dang this is revolutionary, can't believe i didn't know about this sooner